docs/v2/upgrade-guides/2.0.md

```diff
@@ -202,6 +202,10 @@ function get_post( mixed $query = false, array $options = [] );
 
 We deprecated the `$PostClass` parameter. If you want to control the class your post should be instantiated with, use a [Class Map filter](https://timber.github.io/docs/v2/guides/class-maps/#the-post-class-map). Instead, we now have an `$options` array.
 
+#### Changed return type `Timber::get_post()` on failure
+
+`Timber::get_post()` returned `false` if no post could be found in 1.x. Now, that function will return `null` if no post could be found.
```

Comment:
```suggestion
`Timber::get_post()` returned `false` if no post could be found in 1.x. Now, that function will return `null` if no post could be found. We recommend using `null` instead of `false` for when no objects are found to match modern PHP practices.
```